svir/ui/gem_qwebview.py

```diff
@@ -35,10 +35,14 @@
                               QSettings,
                               QByteArray,
                               pyqtSlot,
+                              QUrl,
                               )
 from qgis.PyQt.QtGui import QSizePolicy, QFileDialog
 from svir.utilities.shared import DEBUG, REQUEST_ATTRS
-from svir.utilities.utils import tr
+from svir.utilities.utils import (tr,
+                                  create_progress_message_bar,
+                                  clear_progress_message_bar,
+                                  )
```

Comment:
newline before `tr` and a more compact layout ?

Reply:
It's like this almost everywhere in the plugin. I prefer to keep it as it is, instead of changing it in the whole repository.